tracer/test/snapshots/AspNetCoreIisMvc30Tests.InProcess.WithFF.__path=__statusCode=200.verified.txt

```diff
@@ -30,11 +30,15 @@
       aspnet_core.route: {controller=home}/{action=index}/{id?},
       component: aspnet_core,
       env: integration_tests,
+      http.client_ip: 127.0.0.1,
       http.method: GET,
       http.request.headers.host: localhost:00000,
+      http.route: {controller=home}/{action=index}/{id?},
```

Comment:
true... I guess for now it will be ok, it's optional and just a "best effort"  is required..

Reply:
Going back to a [GitHub Issue](https://github.com/open-telemetry/opentelemetry-dotnet/issues/1661) I linked somewhere in one of my previous comments, the .NET OTEL SDK does the same for `http.route`, so I think it should be fine. Maybe we don't want to include the default value though? Idk

Reply:
I see in the OTEL rfc

> [1]: http.url is usually not readily available on the server side but would have to be assembled in a cumbersome and sometimes lossy process from other information (see e.g. https://github.com/open-telemetry/opentelemetry-python/pull/148). It is thus preferred to supply the raw data that is available.

As I ended up copying aspnetroute / aspnetcore route (ie the route template provided by asp.net (core)) to not block the PR, I guess yes, this will have to be part of another one, but it's true that it's quite costly to resolve all values like in calculateresourcename everytime and maybe not bringing so much value backend side? in the RFC, they remain pretty vague, but they dont seem to be talking about the `template` of the route bu the resolved controller, action and unresolved custom params 🤔